packages/vscode-extension/package.json

```diff
@@ -490,6 +490,11 @@
         "title": "Teams - Validate SPFx Dependencies",
         "enablement": "never"
       },
+      {
+        "command": "fx-extension.validate-local-prerequisites",
```

Comment:
Why add this command in package.json if it'll never show up on UI?

Reply:
It will be only used by `tasks.json`, to replace `fx-extension.validate-dependencies`